### DISCUSSION TITLE
heruko

### DISCUSSION THREAD

**Discussion by pranavsh1408** [anasty17/mirror-leech-telegram-bot#1208] [General]
where is the heruko branch which was earlier present

**Comment by anasty17** [comment]
What is ur problem?
U don't know what will happen after 28 nov!?
No more support for any platform! 
Only docker support.

**Reply by pranavsh1408** [reply]
but i have no issues for paid heruko , due to this removal its not getting deployed on heruoku

**Reply by anasty17** [reply]
Buy vps instead! Heroku will ban my repo again and again and nothing worth.
